src/Host-SDK/Temporal.Worker.Activities/ActivityApi.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Threading.Tasks;
+
+using Temporal.Common.DataModel;
+using Temporal.Serialization;
+
+namespace Temporal.Worker.Activities
+{
+    public class ActivityAPi
+    {
+    }
+
+    // =========== Basic activity abstractions ===========
+
+    public interface IBasicActivity
```

Comment:
Why `Basic` activity? IMO this should just be `IActivity` unless we expect some kind of `Advanced` activity, but I'm not sure what that'd be.

Reply:
Good point.

Reply:
We should also consider a design similar to Java where multiple activities can be colocated in the same class.